DragAndDrop/ViewController.swift

```diff
@@ -20,6 +25,13 @@ class ViewController: UIViewController {
     // Dispose of any resources that can be recreated.
   }
 
+    func reset() {
+        mainView.removeFromSuperview()
+
+        mainView = View(frame: self.view.frame)
```

Comment:
Maybe combine these repeated lines in some way. 
maybe just in a function called 

`initMainView() -> View`
Then you could use it here and for the closure at the top.